### PR TITLE
Added "getAccessMapping" method to AdminExtensionInterface

### PR DESCRIPTION
**Pull request by core23**
Added missing method to interface (See https://github.com/sonata-project/SonataAdminBundle/pull/3297)
This should be scheduled for the next major.

### CODE REVIEW

**Comment by OskarStark** [comment]
Thank you @core23

**Comment by core23** [comment]
Added missing Changelog and Upgrade notes

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 - Add the ``configureActionButtons`` method to the AdminInterface
 - Add the ``configureActionButtons`` method to the AdminExtensionInterface
+- Add the ``getAccessMapping`` method to the AdminExtensionInterface
```
Comment (OskarStark) [inline CHANGELOG.md]: please only one tick instead of two

Comment (core23) [inline CHANGELOG.md]: Is there any difference to two ticks? Thought this is the same. The other code snippets use two ticks.

Comment (OskarStark) [inline CHANGELOG.md]: sorry this is my fault, i thought we use only one tick in this class, but we use two, so everything is fine!

**Comment by core23** [comment]
Rebased

**Comment by OskarStark** [comment]
Thank you @core23

**Pull request merged**